composer.json

```diff
@@ -24,7 +24,7 @@
         }
     ],
     "require": {
-        "php": ">=7.1 <8.1",
+        "php": ">=7.1 <= 8.2",
```

Comment:
You added support of `8.1` and `8.2` by `<= 8.2`, but in changelog, workflow and readme only `8.1`